### PR TITLE
feat: adds code of conduct

### PR DESCRIPTION
**Pull request by daffodil11**

### CODE REVIEW

**Comment by KeiranWDigital** [comment]
Why is the CoC file a markdown file in git? It seems to load right on my local site though

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- This exception specifically does not allow the use of gratuitous sexual images as a device to grab attention or for unnecessary examples.
+
+Thank you
+
+This document was based on the [Sheffield Digital Code of Conduct](https://sheffield.digital/events/meetup-code-of-conduct/), which in turn is based on the work of JSConf, Geek Feminism, confcodeofconduct.com, and inspired by resources provided by Ashe Dryden.
+
+Licensed under a Creative Commons Attribution 3.0 Unported License.
+Version 1
+
+This is version 1 of the CodeWith Code of Conduct. It was last updated by Simon Davey on 31 August 2020. If you think we have made any errors or omissions, please contact us by email (contact@codewith.org.uk) or via the [contact form](contact.html).
```
Comment (dangerworm) [inline code-of-conduct.md]: Could the email address be a mailto: link?

Comment (dangerworm) [inline code-of-conduct.md]: Our contact form doesn't function in a way that lets people send us a message. Perhaps word it as being possible to contact us through one of the methods *on* the contact form?

Comment (daffodil11) [reply code-of-conduct.md]: Addressed.

Comment (daffodil11) [reply code-of-conduct.md]: Addressed.

**Comment by daffodil11** [comment]
> Why is the CoC file a markdown file in git? It seems to load right on my local site though

I'm totally new to Jekyll, so I've been referencing the beginner guide. One of the things it makes a point of is that you can include .md files and it will process them into html (https://jekyllrb.com/docs/step-by-step/04-layouts/). It seemed like a good idea for content like the Code of Conduct, as it would make it easier for non-technical people to contribute to and maintain them in the future, hopefully.

**Comment by dangerworm** [comment]
Fantastic news! Cheers Simon.

**Pull request merged**